draft-ietf-oauth-device-flow.xml

```diff
@@ -415,33 +468,169 @@
+    the user.
+  </t>
+
+  <t>
+    The interval at which the client polls MUST NOT be more frequent than the
+    <spanx style="verb">interval</spanx> parameter returned in the Device Authorization
+    Response (see <xref target="AuthorizationResponse"/>).
+  </t>
+
+  <t>
+  The assumption of this specification is that the secondary device the
+  user is authorizing the request on does not have a way to communicate back
+  to the OAuth client. That only a one-way channel is required makes this flow
+  useful in many scenarios such as a HTML application on TV that can only 
+  make outbound requests. If a return channel were to exist for the chosen user
+  interaction interface, then the device MAY wait until notified on that channel
+  that the user has completed the action before initiating the token request.
+  Such behavior however is outside the scope of this specification.
   </t>
 
   </section>
 
 
     </section> 
 
+    <section anchor="Discovery" title="Discovery Metadata">
+      <t>
+        Support for the device flow MAY be declared in the OAuth 2.0
+        Authorization Server Metadata <xref target="I-D.ietf-oauth-discovery"/>
+        with the following metadata:
+      </t>
+      <t>
+      <list style="hanging"> 
+        <t hangText="device_authorization_endpoint">
+          <vspace/>
+          OPTIONAL.
+          URL of the authorization server's device authorization endpoint 
+          defined in <xref target="AuthorizationRequest"/>.
+        </t>
+      </list>
+      </t>
+    </section>
+
+    <section anchor="Security" title="Security Considerations">
+        <section title="User Code Brute Forcing" anchor="UserCodeBruteForce">
+          <t>
+          Since the user code is typed by the user, the entropy is typically
+          less than would be used for the device code or other OAuth bearer token types.
+          It is therefore recommended that the server rate-limit user code
+          attempts. The user code SHOULD have enough entropy that when combined
+          with rate limiting makes a brute-force attack infeasible.
+          </t>
+        </section>
+        <section title="Device Trustworthiness" anchor="DeviceTrustworthiness">
+          <t>
+          Unlike other native application OAuth 2.0 flows, the device requesting the
+          authorization is not the same as the device that the user grants access
+          from. Thus, signals from the approving user's session and device are
+          not relevant to the trustworthiness of the client device.
+          </t>
+        </section>
+        <section title="Remote Phishing" anchor="RemotePhishing">
+          <t>
+          It is possible for the device flow to be initiated on a device in
+          an attacker's possession. For example, they might send an email
+          instructing the target user to visit the verification URL and
+          enter the user code. To mitigate such an attack it is RECOMMENDED
```

Comment:
s/To mitigate such an attack it is RECOMMENDED during the User Instruction (<xref target="UserInstruction"/>) to inform the user that they are authorizing a device, and to confirm that the device is in their possession.
/ To mitigate such an attack it is RECOMMENDED to inform the user that they are authorizing a device (see <xref target="UserInstruction"/>), and to confirm that the device is in their possession.